src/components/Card/Card.tsx

```diff
@@ -1,14 +1,21 @@
-import React, { ReactNode, ReactElement } from "react";
+import React, { MouseEventHandler, ReactNode, ReactElement } from "react";
 import styled, { css } from "styled-components";
 
 export interface CardProps {
   children: ReactNode;
-  withHover?: boolean;
+  onClick?: MouseEventHandler | null;
```

Comment:
I think this should just be a function

```suggestion
  onClick?: () => void;
```

Reply:
Fixed with bc5b1d608c15454b685ebb57804fc68338d66800